docs/modules/integrations/pages/webhooks/create-and-manage-webhooks.adoc

```diff
@@ -0,0 +1,128 @@
+= Create and Manage Webhook
+:navtitle: Create and Manage Webhooks
+
+Learn how to use Kobiton webhook to send a notification via HTTP to another application when an event occurs.
+
+You can use Kobiton webhooks to notify your app or an internal system when certain events occur in Kobiton. For example, you might use webhooks to alert your remote application when a device has gone offline or becomes available instead of periodically polling Kobiton via REST APIs to determine whether changes have occurred.
+
+Webhooks can be set up by org admins or any role with the *org_setting.modify* permission enabled.
+
+== Supported webhook events
+
+A webhook is triggered to send an HTTP callback when a particular event occurs.
+
+Kobiton currently supports creating device status webhooks for the following events:
+
+. *Online*: from any status, except utilizing, to *online*
+. *Offline*: from any status to offline
+. *Utilizing*: from any status to utilizing
+. *Unplugged*: from any status to unplugged
+
+[NOTE]
+===============================
+Device status webhooks are only available for *Private* and *Local* devices. They cannot be used to report status for *Public* devices.
+===============================
+
+== Create a webhook
+
+In the Kobiton Portal, select your name from the main menu and choose *Settings* from the options.
+
+image:integrations:w1.png[]
+
+Select the **Webhook Settings** tab.
+
+image:integrations:w2.png[]
+
+Select **Create Webhook**.
+
+image:integrations:w3.png[]
+
+Complete the required information as described below:
+
+[options="header"]
+|=======================
+|Parameter|Required|Format|Example|Description

```

Comment:
Example is often the last column - could we change the order of the columns?

Reply:
The order of the column has been changed

Reply:
Verified.